pkg/skaffold/deploy/kpt/kpt.go

```diff
@@ -69,10 +77,64 @@ func NewDeployer(cfg types.Config, labels map[string]string) *Deployer {
 	}
 }
 
+var sanityCheck = versionCheck
+
+// sanityCheck guarantees the kpt and kustomize versions are compatible with skaffold.
+func versionCheck(dir string) error {
```

Comment:
Would it be possible for you to list the error messages in the test case or verify if they look ok manually?

Reply:
Done